### ISSUE TITLE
No failure reason is returned with the response to the send-transaction API

### ISSUE THREAD

**Issue by maciejzaleski** [stratisproject/StratisBitcoinFullNode#2417]
**Request:**
POST /api/Wallet/send-transaction

**Response:**
HTTP 500: Internal Server Error

**StackTrace from console:**
```fail: Microsoft.AspNetCore.Server.Kestrel[13]
      Connection id "0HLHCVSIJNGA8", Request id "0HLHCVSIJNGA8:00000001": An unhandled exception was thrown by the application.
Stratis.Bitcoin.Features.Wallet.WalletException: Can't send transaction: sending transaction requires at least one connection!
   at Stratis.Bitcoin.Features.Wallet.Controllers.WalletController.SendTransaction(SendTransactionRequest request) in C:\Code\InternalTestnet\tmp\StratisBitcoinFullNode\src\Stratis.Bitcoin.Features.Wallet\Controllers\WalletController.cs:line 745
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Cors.Infrastructure.CorsMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpProtocol.ProcessRequests[TContext](IHttpApplication`1 application)```

**Comment by bokobza** [comment]
Fix is in #2413 . The new error returns a 403 with the correct message.

**Comment by maciejzaleski** [comment]
Looks better, I got this message instead of the HTTP 500 status code.
```{
  "errors": [
    {
      "status": 403,
      "message": "Can't send transaction: sending transaction requires at least one connection!",
      "description": ""
    }
  ]
}```
